captionsProcessor.py

```diff
@@ -21,6 +21,7 @@ def main():
     logging.info(f"Retrieving Questions for {config.videoToUse}...")
     questionData = retrieveQuestions(config, topicModeller, videoData)
     questionData.printQuestions()
+    questionData.saveToFile()
```

Comment:
This returns a boolean, I feel you are not doing anything with this. Why return anything from `saveToFile`? since you pretty much logging if save went good or bad.

Reply:
That makes sense. I will remove it for now. If for any reason I need it later I can always add it back.

Reply:
Removed the `return` statements in the latest commit.